src/app/states/facade/chatFacade.ts

```diff
@@ -0,0 +1,26 @@
+import {Observable} from 'rxjs';
+import {Injectable} from '@angular/core';
+import { Store } from '@ngxs/store';
+import {AddMessage, AddMessages} from '../actions/chat.actions';
+import {Messages} from '../../interfaces/messages';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ChatFacade {
+  constructor(
+    private store: Store
+  ) {}
+
+  getAllMessages(): Observable<Messages[]> {
+    return this.store.select(state => state.chat.messages);
```

Comment:
Bestenfalls einen memoized Selector verwenden (https://www.ngxs.io/concepts/select#memoized-selectors):
```suggestion
    return this.store.select(ChatState.messages);
```